ee-9/pom.xml

```diff
@@ -672,6 +673,17 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.jboss.spec.jakarta.xml.ws</groupId>
+                <artifactId>jboss-jakarta-xml-ws-api_3.0_spec</artifactId>
```

Comment:
I miss the license entry for this new artifact. The ee-8-api version has one for `org.jboss.spec.javax.xml.ws:jboss-jaxws-api_2.3_spec`. I would expect the same for the new one.

Reply:
Added. Thanks for review.